packages/big-design/src/components/Worksheet/hooks/useKeyEvents/useKeyEvents.ts

```diff
@@ -83,14 +83,25 @@ export const useKeyEvents = () => {
             navigate({ rowIndex: 0, columnIndex: -1 });
             break;
 
+          case 'Meta':
+            editSelectedCell();
+            break;
+
+          case 'Control':
+            editSelectedCell();
+            break;
+
           default:
             if (
               key !== 'Escape' &&
```

Comment:
Could we do something like this instead of adding more `case` statements?
```suggestion
              !['Meta', 'Control', 'Escape'].includes(key) &&
```
Seems like we don't want to call `editSelectedCell();` when those keys are pressed 🤔

Reply:
The problem is that if we include Meta , and Control, the use is not gonna be able to copy and paste the new content in the cell 👀